src/Components/Navbar.component.js

```diff
@@ -59,8 +65,8 @@ export default class Navigation extends Component {
             </DropdownMenu>
           </Dropdown>
           <Form inline className="ml-auto mr-3">
-            <FormControl type="text" placeholder="Search" className="mr-sm-2" />
-            <Button type="button" color="info">SEARCH</Button>
+            <FormControl type="text" placeholder="Search Term" className="mr-sm-2" id="searchBox" />
+            <Button onClick={() => this.searchClicked()} type="button" color="info">SEARCH</Button>
```

Comment:
`onClick={() => this.searchClicked()}` can be simplified to `onClick={this.searchClicked}`